pennylane/measurements/mid_measure.py

```diff
@@ -82,6 +82,29 @@ def func():
         Python ``not``, ``and``, ``or``, do not work since these do not have dunder methods.
         Instead use ``~``, ``&``, ``|``.
 
+    Mid-circuit measurement results can be processed with the usual measurement functions such as
+    :func:`~.expval`. For QNodes with finite shots, :func:`~.sample` applied to a mid-circuit measurement
+    result will return a binary sequence of samples.
+    See
+    `here <https://docs.pennylane.ai/en/stable/introduction/measurements.html#mid-circuit-measurement-statistics>`_
+    for more details.
```

Comment:
I meant a sphinx cross reference like this:
```suggestion
    See :ref:`here <mid_circuit_measurements>` for more details.
```

Reply:
Oh wait you actually want the mid-circuit-measurements-statistics section. In this case you will need to go to `doc/introduction/measurements.rst`, find the `Mid-circuit measurement statistics` section, and above it, add `.. _mid_circuit_measurements_statistics:`, and in my suggested fix above change `mid_circuit_measurements` to `mid_circuit_measurements_statistics`